### DISCUSSION TITLE
travellings.link 是被墙了吗？

### DISCUSSION THREAD

**Discussion by Liynw** [travellings-link/travellings#1133] [寻求帮助] [answered]
2022.8.5 经鄙人测试，国内上不去但是国外可以。

**Comment by Big-Cake-jpg** [comment]
2022/08/05 23:41，travellings.link 无任何异常现象。坐标山西省晋中市榆次区，DNS 使用 223.5.5.5。

![image](https://user-images.githubusercontent.com/76810494/183112715-0471f251-84c8-4ab8-802d-6153ef9f524e.png)

请检查自己的网络。

**Comment by imjason** [comment, marked as answer]
经检查，部分地区异常，可以使用travellings.cn缓解
